### DISCUSSION TITLE
can not open two file in the same time

### DISCUSSION THREAD

**Discussion by wss29** [eclipse-glsp/glsp#1189] [Q&A]
hi,
   I am not sure anyone discovered this bug
![2023-12-08_11-14-38](https://github.com/eclipse-glsp/glsp/assets/4537809/b2d0e2ae-0202-4ed5-89f2-08da662ccb6d)
when open two tasklist file in node-sever, only one file can add node

**Comment by tortmayr** [comment]
Thanks for reporting this. The bug has already been discovered and is fixed with https://github.com/eclipse-glsp/glsp-client/pull/307.
A minor release (2.1.0) containing this and other bug fixes is planned for next week.
